### DISCUSSION TITLE
Only include dotfiles from home directory

### DISCUSSION THREAD

**Discussion by alirezaalavi87** [borgbase/vorta#1693] [FAQ]
Hi.
I have added /home/me to sources, and tried to exclude anything that is not a dotfile.
I tried:
```
!.*
re:^(?!\.).*
```
and some other ways. It doesn't work.
how can I achieve this?

Ideally, I would like to include `/home/alireza/.*` instead of excluding everything else.
I can achieve this in borg with 
`borg create /repo::archive_name /home/me/.*`

**Comment by real-yfprojects** [comment]
Patterns are matched to the entire path. In case of Vorta backups the path is relative to root `/`.
The correct patterns would therefore be

```
re:^/home/me/[^.]
```

or

```
re:^/home/me/(?!\.)
```
